client/README.md

```diff
@@ -0,0 +1,92 @@
+# etcd/client
+
```

Comment:
Can we add a one-sentence overview of what an etcd client is and how its used. This is in case someone randomly finds this doc.